toolkit/tools/imagecustomizer/docs/cli.md

```diff
@@ -106,6 +106,35 @@ existing RPM repo (such as packages.microsoft.com). Using a cloned repo with
 Disable the base image's installed RPM repos as a source of RPMs during package
 installation.
 
+## --output-pxe-artifacts-dir
+
+Create the specified folder and populate it with the artifacts to be used for
```

Comment:
I've linked to pxe.md in both the configuration page and the cli page.
In the pxe.md, I've explained an overview and what knobs the image customizer provides for the user (and how they fit in the big picture).
I did not explain how to setup a PXE server as this is outside the scope of the image customizers (it can be any distro or even a Windows Server... also, it can use any daemon service to provide the endpoint (apache, etc)). So, I think it'd be better to leave it to other external source on how to set those up.